src/util/codecs.js

```diff
@@ -3,17 +3,8 @@
  * codec strings, or translating codec strings into objects that can be examined.
  */
 
-import videojs from 'video.js';
 import {findBox} from 'mux.js/lib/mp4/probe';
 
-// Default codec parameters if none were provided for video and/or audio
-const defaultCodecs = {
```

Comment:
We no longer return defaults from here, that way we can use mux.js codecs if they exist.